api/alembic/versions/0d46262707af_alter_morecast_forecast_columns_to_float.py

```diff
@@ -0,0 +1,34 @@
+"""Alter morecast_forecast columns to float
+
+Revision ID: 0d46262707af
+Revises: 69cbd7ca2477
+Create Date: 2023-03-14 15:55:31.325373
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0d46262707af'
+down_revision = '69cbd7ca2477'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Do we also want to add the idir username?

Reply:
This migration is just modifying an existing table definition as a couple forecast values are floats instead of ints. The original table definition also includes the create_user (aka IDIR) and create_timestamp.

Reply:
Though we could drop the update_user and update_timestamp fields.

Reply:
Having the create & update for timestamp & user is/was a BC gov-defined best practice for data architecture. I can't remember where it was documented. I had a quick look through my bookmarks and some of the bcgov documentation sites but couldn't find it. 

I agree - the update_ fields could be dropped (no one's gonna stop us from doing us), but it doesn't need to be done now. If it's there now, might as well just keep it in